client-app/pages/product/product.vue

```diff
@@ -36,18 +36,18 @@
             </div>
 
             <div class="mt-7 md:mt-5">
-              <VcButton to="/checkout" class="uppercase px-2 w-full">view cart</VcButton>
+              <VcButton to="/checkout" class="uppercase px-2 w-full" v-t="'pages.product.view_cart_button'"></VcButton>
             </div>
           </ProductPriceBlock>
 
           <!-- Price & Delivery (without variations) -->
           <ProductPriceBlock v-else :product="product">
             <div class="flex items-baseline justify-between text-sm">
-              <div class="font-extrabold text-base">Your price:</div>
+              <div class="font-extrabold text-base" v-t="'pages.product.price_label'"></div>
 
               <div>
                 <VcPriceDisplay :value="product.price?.actual" class="font-extrabold text-green-700" />
-                <span class="font-semibold hidden lg:inline-block ml-1">/ each</span>
+                <span class="font-semibold hidden lg:inline-block ml-1" v-t="'common.suffixes.product_price'"></span>
```

Comment:
Incorrect key name. "Product price" suffix is usually understanded as currency or measure unit, etc. This key should be renamed to "per_item".